packages/components/app/styles/components/alert.scss

```
@@ -15,7 +17,15 @@
   margin-right: 0.75rem; // 12px
 }
 
+.hds-alert__content {
+  flex: 1 1 auto;
+}
+
 .hds-alert__text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 24px; // same as the icon height, to keep text vertically centered in case is a single line
```

(non-blocking) Do we want these all in rem? Ditto to 2px below. I am honestly not sure what we want

```suggestion
  min-height: 1.5rem, // 24px, same as the icon height, to keep text vertically centered in case is a single line
```